constructor-manager-ui/src/constructor_manager_ui/style/base.qss

```diff
@@ -336,6 +336,12 @@ QHeaderView::section {
 
 /* ----------------- Buttons -------------------- */
 
+QPushButton:disabled  {
+  background-color: @background-color13;
+  background-color: @background-selection-color01;
+  background-color: #758193;
```

Comment:
Should here be only the entry for `#758193` while also making it part of the colors constants available?

Reply:
Yes, we can do that